### PR TITLE
Tag images with short commit SHA

### PR DESCRIPTION
**Pull request by geigerzaehler**
Tag images with short commit SHA

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,13 +21,6 @@ docker run \
   gcr.io/opensourcecoin/ipfs-test-network
 ```
 
-To wait until the nodes have connected to each other run
-```
-docker run ipfs-test-network /app/wait-until-ready
-```
-The command will timeout after 10 seconds and exit with status code 1.
-
-
```
Comment (xla) [inline README.md]: This delete seems unrelated to the change of this PR.

**Review by xla — APPROVED** [review]
:+1: :shipit: :dancer:

File: README.md (hunk 1 above)
Comment (geigerzaehler) [reply README.md]: Rebase issue
